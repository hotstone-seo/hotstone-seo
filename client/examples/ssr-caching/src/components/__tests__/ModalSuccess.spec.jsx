import React from 'react';
import ModalSuccess from '../ModalSuccess.jsx';

test('Should render ModalSuccess correctly', () => {
  const tree = enzyme.mount(<ModalSuccess />);

  console.log(tree.debug());
  expect(tree.find('div.modal-success').length).toEqual(1);
  expect(tree).toMatchSnapshot();
  tree.unmount();
  expect(tree.find('div.modal-success').length).toEqual(0);
});
