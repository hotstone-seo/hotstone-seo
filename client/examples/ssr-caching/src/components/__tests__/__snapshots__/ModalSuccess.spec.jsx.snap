// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ModalSuccess correctly 1`] = `
<ModalSuccess>
  <Modal
    className="modal-success"
    footer={
      <div
        className="row"
      >
        <div
          className="col-12"
        >
          <a
            className="btn large block"
            href="https://m.renan.tiket.com/myorder"
          >
            Back to My Order
          </a>
        </div>
      </div>
    }
    style={
      Object {
        "maxWidth": "80%",
      }
    }
  >
    <div
      className="modal"
      style={Object {}}
    >
      <div
        className="modal-content modal-success"
        style={
          Object {
            "maxWidth": "80%",
          }
        }
      >
        <div
          className="modal-body"
        >
          Thank you for booking at tiket.com. You can still continue your payment from My Order
        </div>
        <div
          className="modal-footer"
        >
          <div
            className="row"
          >
            <div
              className="col-12"
            >
              <a
                className="btn large block"
                href="https://m.renan.tiket.com/myorder"
              >
                Back to My Order
              </a>
            </div>
          </div>
        </div>
      </div>
    </div>
  </Modal>
</ModalSuccess>
`;
